pkgs/by-name/re/relnotes/package.nix

```
@@ -0,0 +1,46 @@
+{
+  darwin,
+  fetchFromGitHub,
+  lib,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "relnotes";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "EmbarkStudios";
+    repo = "relnotes";
+    rev = "v${version}";
+    hash = "sha256-3Jv5GTB528wZRpQlJtoqqTEL0OA9Cf0htgZbS5H4Xrw=";
+  };
+
+  cargoHash = "sha256-earxhVkEhHZipvSHTKzoINIq0/PGyt6CY35mv+ksv1E=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs =
+    lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ]
+    ++ lib.optionals stdenv.isLinux [ (lib.getDev openssl) ];
```

```suggestion
  buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ (lib.getDev openssl) ];
```
I think the Darwin stuff is included automatically by now.